src/users/containers/edit-user-permissions/edit-user-permission.component.ts

```diff
@@ -115,6 +120,7 @@ import { checkboxesBeCheckedValidator } from '../../../custom-validators/checkbo
   ngOnDestroy() {
     this.unsubscribe(this.userSubscription);
     this.unsubscribe(this.dependanciesSubscription);
+    this.unsubscribe(this.editPermissionServerErrorSubscription);
```

Comment:
Maybe check if it exists first? All three subscriptions... :)

Reply:
Please unsubscribe method